hypertrace-view-generator/hypertrace-view-creator/src/main/resources/configs/log-event-view/application.conf

```diff
@@ -0,0 +1,55 @@
+service.name = view-creator
+service.admin.port = 8099
+
+main.class = org.hypertrace.core.viewcreator.ViewCreatorLauncher
+tool.class = org.hypertrace.core.viewcreator.pinot.PinotTableCreationTool
+
+view.name = logEventView
+view.output.schema.class = org.hypertrace.viewgenerator.api.LogEventView
+view.output.schema.url = "host:port/myView"
+
+kafka.brokerAddress = "bootstrap:9092"
+kafka.topicName = log-event-view
+kafka.partitions = 1
+kafka.replicationFactor = 1
+pinot.controllerHost = pinot-controller
+pinot.controllerPort = 9000
+pinot.timeColumn = timestamp_nanos
+pinot.timeUnit = NANOSECONDS
+pinot.dimensionColumns = [tenant_id, span_id, trace_id, timestamp_nanos, attributes]
+pinot.columnsMaxLength={}
+pinot.metricColumns = []
+pinot.invertedIndexColumns= []
+pinot.rangeIndexColumns = [timestamp_nanos]
+pinot.bloomFilterColumns = []
+pinot.noDictionaryColumns = []
```

Comment:
we probably shouldn't use dictionary encoding for log content ?

Reply:
Yes, make sense. we are planning to use a text search index. But, yes, we should remove it from default too.